packages/core/src/node/messaging/connection.ts

```diff
@@ -44,6 +44,36 @@ export function openSocket(options: IServerOptions, onOpen: OnOpen): void {
         noServer: true,
         perMessageDeflate: false
     });
+    interface ExtWebSocket extends ws {
+        isAlive: boolean;
+    }
+
+    wss.on('connection', (ws: ws) => {
+
+        const extWs = ws as ExtWebSocket;
+        extWs.isAlive = true;
+
+    });
+
+    setInterval(() => {
+        wss.clients.forEach((ws: ws) => {
+
+            const extWs = ws as ExtWebSocket;
+
+            ws.on('pong', () => {
```

Comment:
It will install a new listener each 30s, please move it in `ws.on('connection', ...` to install it once, as here; https://github.com/websockets/ws#how-to-detect-and-close-broken-connections

Reply:
Will do.